libs/net/dal/Services/ReportInstanceService.cs

```diff
@@ -51,7 +53,7 @@ public IEnumerable<ReportInstance> FindInstancesForReportId(int reportId, int? o
         if (ownerId.HasValue)
             query = query.Where(ri => ri.OwnerId == ownerId);
 
-        query = query.OrderByDescending(ri => ri.Id);
+        query = query.OrderByDescending(ri => ri.Id).Skip(skip).Take(take);
```

Comment:
Allow us to page through instances.  This was for some reason resulting in an OOM issue when returning hundreds of records.  We will never need to go back through history that far.  At some point we should clean these out.